### ISSUE TITLE
Add "enable" cli command to undo "disable"

### ISSUE THREAD

**Issue by phocks** [superseriousbusiness/gotosocial#2791]
Maybe I'm missing something. I disabled an account with admin disable. Now I just want to enable it again.......

https://docs.gotosocial.org/en/latest/admin/cli/#gotosocial-admin-account-disable

**Comment by tsmethurst** [comment]
Added an `enable` command in https://github.com/superseriousbusiness/gotosocial/pull/2792, which will be included in 0.15.0. In the meantime, you can reenable your user by running the following SQL query (and then restarting your GoToSocial instance):

```
UPDATE users
SET disabled = false
WHERE users.account_id IN (
	SELECT id FROM accounts WHERE accounts.domain IS NULL AND accounts.username = 'SOME_USERNAME_HERE'
);
```

Replace `SOME_USERNAME_HERE` with the username of the account you're trying to reenable.

**Comment by phocks** [comment]
Thank you so much!

**Comment by tsmethurst** [comment]
I'll close this now, since the work is completed and will be included in 0.15.0 :)
